daemon/container_operations.go

```diff
@@ -1140,15 +1138,6 @@ func (daemon *Daemon) DeactivateContainerServiceBinding(containerName string) er
 	return sb.DisableService()
 }
 
-func getNetworkID(name string, endpointSettings *networktypes.EndpointSettings) string {
-	// We only want to prefer NetworkID for user defined networks.
-	// For systems like bridge, none, etc. the name is preferred (otherwise restart may cause issues)
```

Comment:
Do the built-in network have a stable ID? (curious if they are re-created at restart, and if that was related as well)

Reply:
Built-in networks are recreated if no containers are attached to them. Thus, they don't have a stable ID but that wasn't an issue here since the networks weren't recreated in that case.

Reply:
Right, so if there's an existing container (either _created_ or _running_) that's attached, the network won't be re-created, so the ID is stable, correct?

Is this code-path hit by Swarm, or not hit? (Just making sure, ISTR some subtle things with swarm pinning to an ID, and that ID of course being different between nodes or dynamically deployed, and IDs could not be used for that reason).